source/_integrations/plugwise.markdown

```diff
@@ -229,3 +229,14 @@ Stretch (power switches):
 
  - v3.x
  - v2.x
+
+### Thermostat target temerature step size
+
+By default, the step size for Plugwise thermostats is set to 0.1. However, the hardware buttons on those thermostats use a step size of 0.5. This means that, if you press `+` on the thermostat in the Home Assistant UI, it will increase the target temperature with 0.1, but if you press `+` on the device itself, it will increase the target temperature with 0.5. If you don't like this, you can change the step size by adding the following snippet to your `configuration.yaml` _for each thermostat_:
+
+```yaml
+homeassistant:
+  customize:
+    climate.your_thermostat_name:
+      target_temp_step: 0.5
+```
```

Comment:
This should be used for that, nor do we recommend the use of customize in general, as this overrides/injects things directly into the state machine (and may cause problems of its own).

For that reason, I'm going to friendly decline this addition to the documentation.

../Frenck

Reply:
I didn't know that. Now I understand why I didn't find any information on it. I guess that's a +1 for my [feature request](https://community.home-assistant.io/t/make-target-temperature-step-size-configurable-in-plugwise-integration/717781) then? 😉